WalletWasabi.Fluent/ViewModels/Wallets/Send/TransactionPreviewViewModel.cs

```diff
@@ -275,7 +275,7 @@ private async Task<bool> InitialiseTransactionAsync()
 		return true;
 	}
 
-	private async Task<BuildTransactionResult?> BuildTransactionAsync(BuildTransactionReason reason)
+	private async Task<BuildTransactionResult?> BuildTransactionAsync()
```

Comment:
The fact that this method is recursive but it doesn't pass any parameter between recursive calls feels unsettling.

Reply:
https://github.com/zkSNACKs/WalletWasabi/pull/9508#discussion_r1049297493